vignettes/estimate_ascertainment.Rmd

```diff
@@ -20,128 +20,100 @@ editor_options:
+
+For example, during the recent COVID-19 pandemic, the U.K. did not roll out widespread testing until around May 2020.
+This means that during March and April, infections were heavily under-ascertained.
+We focus on this period within this vignette, as it illustrates the use for this analysis pipeline well. -->
+
+This vignette shows how to use the `estimate_reporting()` function in _cfr_ to estimate the proportion of cases, infections or hospitalisations ascertained, and consequently, the extent of under-ascertainment or under-reporting.
+
+First load _cfr_ and packages to access and plot data.
 
 ```{r, message = FALSE, warning=FALSE, eval = TRUE}
+# load {cfr} and data packages
 library(cfr)
 library(epiparameter)
 library(covidregionaldata)
-```
 
-and the four optional packages, to run the multiple examples in the last 
-section of this vignette:
-
-```{r}
+# packages to wrangle and plot data
 library(dplyr)
 library(tidyr)
 library(purrr)
 library(scales)
-library(ggplot2)
 library(forcats)
+library(ggplot2)
 ```
 
-# Motivation
+## Ascertainment for the Covid-19 pandemic in the U.K.
 
-There are several reasons why cases, infections or hospitalisations and 
-deaths might be under-ascertained during an outbreak of an infectious disease. 
-Some examples include:
+The function `estimate_reporting()` from the _cfr_ package estimates the proportion of cases, infections, hospitalisations -- or whichever proxy for infections is provided -- which have been ascertained.
 
-* Testing capacity or criteria is not sufficient to identify all infections.
+<!-- The method used within this function extends the methods outlined in the previous vignettes about estimating the severity during an ongoing outbreak and measuring how the severity changes over time. -->
 
-* Symptom-based testing rather than sampling at random in the 
-population. This can lead to biases due to some infections either being 
-asymptomatic or having a pre-symptomatic phase.
+The methods are based on the @nishiura2009 to estimate severity, and are extended by combining the resulting severity estimates with an assumed baseline severity estimate.
 
-* Changes in case definition over time.
+The proportion of cases, infections or other quantity provided that have been ascertained is given by the ratio of the (assumed) true baseline severity estimate, to the delay-adjusted severity estimate.
 
-For example, during the recent COVID-19 pandemic, the U.K. did not roll
-out widespread testing until around May 2020. This means that during March and 
-April, infections were heavily under-ascertained. We focus on this period
-within this vignette, as it illustrates the use for this analysis pipeline well.
+The delay-adjusted severity estimates can be calculated using either the `estimate_static()` or the `estimate_time_varying()` functions. 
 
-# Methods
+<!-- # Example with data from the ongoing COVID-19 pandemic in the U.K.
 
-## Data required
```

Comment:
To promote the usability of the pipelines and the interoperability between tools, it'd be good to mention explicitly what format of data is needed for each function

Reply:
This is now added in the section "What we have". Will add to all vignettes.